modules/contact-form/grunion-contact-form.php

```diff
@@ -2118,6 +2118,19 @@ function process_submission() {
 		 */
 		$message = apply_filters( 'contact_form_message', $message );
 
+		// Add HTML tags after the `contact_form_message` filter to preserve back-compat
+		$message = str_replace( "\t", '', sprintf(
+			"<!doctype html>
```

Comment:
If there's some code in a third-party plugin already wrapping the message in something similar to this, in the prior filter, should we test for `<!doctype` or `<html` in the message before wrapping it ourselves?  Or am I overthinking this and it would be relatively harmless to double-wrap?